server/src/domain/user/dto/update-user.dto.ts

```diff
@@ -49,4 +50,13 @@ export class UpdateUserDto {
   @Optional()
   @IsBoolean()
   memoriesEnabled?: boolean;
+
+  @Optional()
+  @IsString()
+  @IsNotEmpty()
+  avatarColor?: UserAvatarColor;
+
+  @Optional()
+  @IsString()
+  profileImagePath?: string;
```

Comment:
Why is this added here?

Reply:
A rest of my previous implementation, I have reverted it